service-workers/cache-storage/window/cache-put.https.html

```diff
@@ -1,22 +0,0 @@
-<!DOCTYPE html>
-<title>Cache Storage: Cache.put</title>
-<link rel="help" href="https://w3c.github.io/ServiceWorker/#cache-put">
-<meta name="timeout" content="long">
-<script src="/resources/testharness.js"></script>
-<script src="/resources/testharnessreport.js"></script>
-<script src="/common/get-host-info.sub.js"></script>
-<script src="../resources/test-helpers.js"></script>
-<script src="../script-tests/cache-put.js"></script>
-<script>
-cache_test(async (cache) => {
```

Comment:
Moved this test into the `.any.js` file. It is not window specific AFAICT (`FormData` is available in all contexts that `fetch` is).